lldb/docs/resources/lldbgdbremote.md

```diff
@@ -1998,6 +1998,19 @@ threads (live system debug) / cores (JTAG) in your program have
 stopped and allows LLDB to display and control your program
 correctly.
 
+## qWasmCallStack
+
+Get the Wasm callback for the given thread id. This returns a hex-encoding list
+of 64-bit addresses for the frame PCs. To match the Wasm specification, the
```

Comment:
If I didn't write debug software for a living, this phrasing would make no sense to me. How about:
```
This returns a hex-encoded list of PC values, one for each frame of the call stack.
```
I get what you meant but a pedant might think it meant that each one was the *location* of a value, that itself was the value of a PC.